quarkus/tests/integration/src/test/java/org/keycloak/it/cli/StartCommandTest.java

```diff
@@ -39,13 +40,15 @@ void failNoTls(LaunchResult result) {
 
     @Test
     @Launch({ "--profile=dev", "start" })
+    @WithLegacyStoreOnly
```

Comment:
It's not possible to use the CHM for the `start` command as it requires real prod DB, right?

Reply:
No, It seems is possible to use "storage=chm" in `start` command If you set the `--auto-build` option (which is set by default since Version 18).